src/vorta/utils.py

```diff
@@ -208,6 +208,18 @@ def set_tray_icon(tray, active=False):
     tray.setIcon(icon)
 
 
+def detect_use_light_icon():
```

Comment:
How about calling it `uses_dark_theme()` or something?

This accidentally makes sense for the setting `use_light_icon` as well. `uses_dark_theme() = use_light_icon = True`

Reply:
Sure, I renamed it to uses_dark_mode